Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -940,11 +700,10 @@ - (void)playNativeAd
     if ( !self.vungleNativeAd )
     {
         [self.nativeAdDelegate didFailToLoadNativeAdWithError: MAAdapterError.noFill];
-        return;
```

Comment:
I think we might need to return when `self.vungleNativeAd` is nil. Thanks a lot. :)